### PR TITLE
Hotel price update

### PR DESCRIPTION
**Pull request by leoli001**
Update bellboy hotel price endpoint

### CODE REVIEW

File: source/includes/_hotels.md [hunk 1]
```diff
@@ -760,10 +760,11 @@ Is a regular practice following the user flow first of all search for prices usi
 
 *API endpoint*
 
-`GET /v1/prices/hotel/{hotel_id}`
+`GET /hbe-bellboy/v/prices/hotel/{hotel_id}`
```
Comment (esaezgil) [inline source/includes/_hotels.md]: I think this `/hbe-bellboy/v/` is wrong. We do not have a `/v/` versioned endpoint

File: source/includes/_hotels.md [hunk 2]
```diff
@@ -760,10 +760,11 @@ Is a regular practice following the user flow first of all search for prices usi
...
+`GET /hbe-bellboy/v2/prices/hotel/{hotel_ids}`
 
 ```shell
-GET "https://gateway.skyscanner.net/hotels/v1/prices/hotel/{hotel_id}
+GET "https://gateway.skyscanner.net/hbe-bellboy/v2/prices/hotel/{hotel_id}
```
Comment (esaezgil) [inline source/includes/_hotels.md]: not sure whether `v2` is publicly available or just for certain partners. I will confirm with Toni Marques

File: source/includes/_hotels.md (hunk 1 above)
Comment (esaezgil) [reply source/includes/_hotels.md]: Additionally, we should keep the `/hotel/` path, as you can see in our drone: https://gitlab.skyscannertools.net/hotels-backend/bellboy/blob/master/.drone.yml#L173 we are publishing the API both as `/hbe-bellboy/` for internal clients and `/hotels/` for B2B

File: source/includes/_hotels.md (hunk 2 above)
Comment (esaezgil) [reply source/includes/_hotels.md]: So after speaking with Toni we have agreed the following:

- Both v1 and v2 prices endpoint should be documented.
- Simply changing `v1` for `v2` is wrong as the API response is different, you can check it in our raml: https://gitlab.skyscannertools.net/hotels-backend/bellboy/blob/master/docs/raml/bellboy.raml#L474

Please fix this and add the proper documentation for prices `v2` which is a different use case in which multiple hotels can be requested at the same time and also the response of the API is different.

**Comment by scottandersonSkyscanner** [comment]
closing this as it is a year old.

**Pull request closed without merging**